### PR TITLE
[BLCKCHN-192]  Add dynamic rendering of token information

### PR DESCRIPTION
**Pull request by yeehan-crypto-com**
https://mcoproduct.atlassian.net/browse/BLCKCHN-192

#### Send transaction
Screenshot 2024-10-30 at 2 16 36 PM
#### Wrap
Screenshot 2024-10-30 at 2 18 58 PM
#### Swap
Screenshot 2024-10-30 at 3 09 33 PM

### CODE REVIEW

File: src/constants/global.constants.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  388: '0x4E792B8c9bcB9E200C3713810C4D6eA8C4230E7c',
+  240: '0x9EB4db2E31259444c5C2123bec8B17a510C4c72B',
+};
+
+export const WRAPPED_ADDRESS_MAPPING: Record<number, string> = {
+  25: '0x5C7F8A570d578ED84E63fdFA7b1eE72dEae1AE23', // Mainnet WCRO
+  338: '0x6a3173618859c7cd40faf6921b5e9eb6a76f1fd4', // Testnet WCRO
+  388: '0xC1bF55EE54E16229d9b369a5502Bfe5fC9F20b6d', // Mainnet wzkCRO
+  240: '0xeD73b53197189BE3Ff978069cf30eBc28a8B5837', // Testnet wzkCRO
+};
```
Comment (leejw51crypto) [inline src/constants/global.constants.ts:15]: just suggestion, can we check validity of contract,
If address is wrong, show more detailed error message like

`check contract address` ?

File: src/constants/chain.constants.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    explorers: [
+      {
+        name: 'Cronos Explorer',
+        url: 'https://explorer.cronos.org',
+        standard: 'none',
+      },
+    ],
+  },
+  {
+    name: 'Cronos Mainnet',
```
Comment (leejw51crypto) [inline src/constants/chain.constants.ts]: seems duplicate

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: src/constants/global.constants.ts (hunk 1 above)
Comment (leejw51crypto) [reply src/constants/global.constants.ts:15]: can be done as another issue

File: src/constants/chain.constants.ts (hunk 2 above)
Comment (yeehan-crypto-com) [reply src/constants/chain.constants.ts]: oh yes it is, let me remove it

File: src/constants/global.constants.ts (hunk 1 above)
Comment (yeehan-crypto-com) [reply src/constants/global.constants.ts:15]: Good point, maybe it can be an optimisation point for signer-app -> optimise error handling for frontend

**Pull request merged**